### PR TITLE
formatting and a fix for the associated webpage

### PR DESCRIPTION
**Pull request by todrobbins**
Updated GitHub Pages site to new URL: http://cellar-wg.github.io/matroska-specification/

### CODE REVIEW

**Comment by todrobbins** [comment]
Also, I don't have permissions, but the link should likely be added to the repo description as well for ease of access. 👍

File: README.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ This repository holds content related to the official Matroska specification and
...
+To create local copies of the RFC in `.txt`, `.md`, and `.html` format, run `make`.
 
 To create a local copy of the website, run `make website`. This command has additional dependencies. See the "About this site" section below for installing and local rendering.
 
 To remove generated specifications, run `make clean`.
 
 ## About this site
 
-This site runs on [GitHub Pages](https://pages.github.com/) powered by [Jekyll](https://github.com/jekyll/jekyll/blob/master/README.markdown). Approved changes made to the `master` branch of this repository can be viewed instantly on the [associated webpage](http://matroska-org.github.io/matroska-specification/).
+This site runs on [GitHub Pages](https://pages.github.com/) powered by [Jekyll](https://github.com/jekyll/jekyll/blob/master/README.markdown). Approved changes made to the `master` branch of this repository can be viewed instantly on the [associated webpage](http://cellar-wg.github.io/matroska-specification/).
```
Comment (JeromeMartinez) [inline README.md]: I suggest to move to `https` at the same time.

**Comment by robUx4** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (todrobbins) [reply README.md]: I agree.

**Comment by dericed** [comment]
thx @todrobbins!

**Comment by todrobbins** [comment]
No problem! Also, a reminder to add http://cellar-wg.github.io/matroska-specification/ to the repo description. 😄

**Comment by dericed** [comment]
done

**Comment by todrobbins** [comment]
@dericed not sure what's up but GitHub is not recognizing the link:

Screen Shot 2019-04-26 at 10 03 36 AM

There are two fields: description and website, the link should go in the website field to become clickable, I believe. Sorry to pester!

**Comment by JeromeMartinez** [comment]
> the link should go in the website field to become clickable, I believe. 

Looks like a bug of GitHub when description is empty. I added a description and now the link is clickable.

**Comment by todrobbins** [comment]
Ah, good to know @JeromeMartinez!

File: README.md (hunk 1 above)
Comment (robUx4) [reply README.md]: Done

**Pull request merged**